osu.Game/Screens/Ranking/Statistics/StatisticItem.cs

```diff
@@ -48,7 +49,7 @@ public StatisticItem([NotNull] string name, [NotNull] Drawable content, [CanBeNu
         /// <param name="createContent">A function returning the <see cref="Drawable"/> content to be displayed.</param>
         /// <param name="requiresHitEvents">Whether this item requires hit events. If true, <see cref="CreateContent"/> will not be called if no hit events are available.</param>
         /// <param name="dimension">The <see cref="Dimension"/> of this item. This can be thought of as the column dimension of an encompassing <see cref="GridContainer"/>.</param>
-        public StatisticItem([NotNull] string name, [NotNull] Func<Drawable> createContent, bool requiresHitEvents = false, [CanBeNull] Dimension dimension = null)
+        public StatisticItem(LocalisableString name, [NotNull] Func<Drawable> createContent, bool requiresHitEvents = false, [CanBeNull] Dimension dimension = null)
```

Comment:
Why isn't this

```suggestion
        public StatisticItem(LocalisableString? name, [NotNull] Func<Drawable> createContent, bool requiresHitEvents = false, [CanBeNull] Dimension dimension = null)
```

And no, this isn't a NRT issue or anything, because `LocalisableString` is a struct. It's really weird to have a nullable name field, then one obsoleted ctor, and then another that receives the name, but the name isn't actually optional. It looks like a bug/misannotation.

Also, you will want to chase down all `StatisticItem`s that pass `string.Empty` as the name and update them to pass `null` instead, because otherwise, this breakage happens:

| master | this PR |
| :-: | :-: |
| ![osu_2022-08-14_16-45-28](https://user-images.githubusercontent.com/20418176/184542338-b460b3d4-303f-4ca1-9436-242226505435.jpg) | ![osu_2022-08-14_16-46-52](https://user-images.githubusercontent.com/20418176/184542329-c8daa5ec-2d8f-4dc9-b69a-79af1a685b97.jpg) |

Reply:
The fix looks ok, but this needs to be listed in the OP as a breaking change so that consumers know they also need to change empty strings to `null`.